src/runtime_src/core/pcie/tools/xbmgmt/firmware_image.cpp

```diff
@@ -264,7 +264,7 @@ DSAInfo::DSAInfo(const std::string& filename, uint64_t ts, const std::string& id
         }
         // For 2RP platform, only UUIDs are provided
         //timestamp = ap->m_header.m_featureRomTimeStamp;
-        hasFlashImage = (xclbin::get_axlf_section(ap, MCS) != nullptr) ||
+        hasFlashImage = (xclbin::get_axlf_section(ap, MCS) != nullptr) || (ap->m_header.m_mode == XCLBIN_FLAT) ||
```

Comment:
Can we please make this change in xbmgmt2/flash/firmware_image.cpp as well?

Reply:
Added the fix in xmgmt2. Please review.